internal/http/services/owncloud/ocdav/dav.go

```diff
@@ -286,6 +328,16 @@ func (h *DavHandler) Handler(s *svc) http.Handler {
 				return
 			}
 
+			if userExists {
+				// Build new context without an authenticated user for the public link
```

Comment:
Please add some more detail here, why we need to rebuild the context without the user. Just to help other people looking at the code.